pymc/smc/sampling.py

```diff
@@ -406,3 +415,25 @@ def run_chains(chains, progressbar, params, random_seed, kernel_kwargs, cores):
                         progress.update(status=f"Stage: {stage} Beta: {beta:.3f}", task_id=task_id)
 
         return tuple(cloudpickle.loads(r.result()) for r in futures)
+
+
+def _find_custom_methods(model):
+    custom_methods = {}
+    for rv in model.free_RVs + model.observed_RVs:
+        cls = rv.owner.op.__class__
+        if hasattr(cls, "_random_fn"):
```

Comment:
The way I'm checking if the rv is from `CustomDist` feels kind dirty. Is there a better way? I couldn't come up with anything better after `_print_name` was changed in aa679f3

Reply:
You can check that `rv.owner.op` is an instance of `CustomDistRV` (or whatever it is called)